inner/src/lib.rs

```diff
@@ -94,6 +116,12 @@ mod store {
         value_type: proc_macro2::TokenStream,
     ) -> (proc_macro2::TokenStream, proc_macro2::TokenStream) {
         // This is the unbounded default.
+        if let Some(hasher) = &_options.custom_hasher {
```

Comment:
just a small style preference: would you mind doing this as if/else, instead of early return?

Reply:
Fixed in new commit